lib/StripeMethod.js

```diff
@@ -20,19 +22,24 @@ const makeAutoPaginationMethods = require('./autoPagination')
  * @param [spec.host] Hostname for the request.
  */
 function stripeMethod(spec) {
+  if (!!spec.path && !!spec.fullPath) {
```

Comment:
super-nit: let's rule out empty strings too, to forbid
```javascript
{
  fullPath: '',
  path: '',
}
```

```suggestion
  if (!!(spec.path || spec.path === '') && !!(spec.fullPath || spec.path === '')) {
```

Reply:
Resolved in https://github.com/stripe/stripe-node/pull/1200/files.